src/sharing/ShareToStream.js

```diff
@@ -150,64 +94,45 @@ class ShareToStream extends React.Component<Props, State> {
 
   render() {
     const { sharedData } = this.props.route.params;
-    const { stream, topic, message, isStreamFocused, isTopicFocused, sending } = this.state;
+    const { stream, topic, isStreamFocused, isTopicFocused } = this.state;
     const narrow = streamNarrow(stream);
+    const sendTo = { stream, topic, type: 'stream' };
 
     return (
-      <>
-        <ScrollView style={styles.wrapper} keyboardShouldPersistTaps="always" nestedScrollEnabled>
-          <View style={styles.container}>
-            {sharedData.type === 'image' && (
-              <Image source={{ uri: sharedData.sharedImageUrl }} style={styles.imagePreview} />
-            )}
-            <AnimatedScaleComponent visible={isStreamFocused}>
```

Comment:
I couldn't notice any change, but couldn't speculate that this could have any potential changes either, (I still struggle with css :sweat_smile:).

Regardless, I tested this on a tablet as well to see if everything is as it should be and it looked like before.
The relevant documentation would be https://www.w3.org/TR/css-flexbox-1/#flex-initial
which makes me feel that It can be a problem in small screens? I tested it on 480x800 emulator and it looked fine as well, not sure which case it will fail

Reply:
Cool, sounds good.

As described at https://github.com/zulip/zulip-mobile/pull/4757#issuecomment-881039196 , for this particular refactor, if this sort of thing seems to work in the new version, then that's good enough. If we find some issue in the layout later, we can fix it then.